var/spack/repos/builtin/packages/openpmd-api/package.py

```diff
@@ -172,17 +172,9 @@ def check(self):
             # later tests
             ctest("--output-on-failure", "-j1")
 
-    def test(self):
-        """Perform smoke tests on the installed package."""
-        exes = ["openpmd-ls"]  # in 0.11.1+
-        for exe in exes:
-            spec_vers_str = "{0}".format(self.spec.version)
-            reason = "test version of {0} is {1}".format(exe, spec_vers_str)
-            self.run_test(
-                exe,
-                ["--version"],
-                [spec_vers_str],
-                installed=True,
-                purpose=reason,
-                skip_missing=False,
-            )
+    def test_run_openpmd_ls(self):
+        """Test if openpmd-ls runs"""
+        if self.spec.satisfies("0.11.0"):
```

Comment:
This should be a version range and starting with the one in the comment.

There's more in the docs on version ranges at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges.